app/models/reservation.rb

```diff
@@ -103,6 +103,25 @@ class Reservation < ActiveRecord::Base
       .select('reservations.*, equipment_models.category_id as category_id')
   }
 
+  # for status modifying jobs
+  def self.deletable_missed
```

Comment:
I think the only one that should be a query object, if any, is `deletable_missed`, because it isn't just a scope chain.

I would rather leave these in the model for a few reasons:
* Moving the last of the `where` calls to scopes
* Keeping the interactions of all of the jobs with `Reservation` the same -- they all call one query
* Avoiding message chains
* Much cleaner code outside of the Reservation class. As cool as the `inject` to chain queries with arguments from an array was, it was unnecessary and required an odd format for the parameters. Now `ReservationJob#collection` takes a single symbol as an argument.

If we decide to keep things in the model, I'll move `deletable_missed` to a query object and the others to proper scopes rather than class methods